docs/admin/node.md

```diff
@@ -258,6 +258,29 @@ Set the `cpu` and `memory` values to the amount of resources you want to reserve
 Place the file in the manifest directory (`--config=DIR` flag of kubelet).  Do this
 on each kubelet where you want to reserve resources.
 
+### Horizontal auto-scaling of nodes
+
+If you are using GCE, you can configure your cluster so that the number of nodes will be automatically scaled based on their utilization.
+Before setting-up cluster by ```kube-up.sh```, you can set ```KUBE_ENABLE_NODE_AUTOSCALER``` environment variable to ```true``` and export it.
+The script will create an autoscaler for the instance group managing your nodes.
+
+The autoscaler will try to  maintain the average utilization of nodes within the cluster close to the target value.
+The target value can be configured by ```KUBE_TARGET_NODE_UTILIZATION``` environment variable (default: 0.7).
+The node utilization is the total node's cpu/memory usage (OS + k8s + user load) divided by the node's capacity.
```

Comment:
Maybe say a little more about how you turn this into a single number -- e.g. is it
(CPU load/CPU capacity + memory load/memory capacity)/2 ?